docs/langref.md

```diff
@@ -162,6 +162,31 @@ Example := Rectangle {
 }
 ```
 
+You can annotate the properties with a qualifier that specifies how the property can be read and written.
+
+ * **`private`**: The property can only be accessed from within the component.
+ * **`in`**: The property is an input, it can be set and modified by the user of this component,
+   for example through bindings or by assignment in callbacks.
+   The component can provide a default binding, but it cannot overwrite it by a `background = #abc`
+   assignment
```

Comment:
The `background` assignment makes sense in the context of the blog post, but here IMO it's not so good to use the name without context.

```suggestion
   The component can provide a default binding, but it cannot overwrite it by
   assignment
```